package.json

```diff
@@ -68,13 +68,14 @@
     "sinon": "~1.12.1",
     "sinon-chai": "^2.8.0",
     "style-loader": "^0.12.4",
+    "substyle": "^3.0.1",
```

Comment:
Why is `substyle` included in the `dependencies` and `devDependencies`? Shouldn't one be enough?